imjoy/imjoyPluginEngine.py

```diff
@@ -847,7 +847,6 @@ def stop_callback(success, message):
             if "aborting" in plugin_info:
                 plugin_info["aborting"].set_result(success)
             message = str(message or "")
-            message = message[:100] + (message[100:] and "..")
```

Comment:
I'd like to remove this message limiting. It was really hard to troubleshoot until I removed this, since the stack trace was incomplete.

Reply:
I think it should be safe to remove this limit.

Reply:
Thanks!